test/y2partitioner/widgets/pages/nfs_mounts_test.rb

```diff
@@ -28,6 +28,8 @@
 describe Y2Partitioner::Widgets::Pages::NfsMounts do
   before do
     devicegraph_stub("nfs1.xml")
+    # Workaround. Will not be needed as soon as the xml format includes MountPoint#mount_type
```

Comment:
I will work on it right now (proper support for that attribute when read from XML). But I didn't want to wait another 12+ hours to have a testable docker image and merge this.